docs/customization/custom-scripts.md

```diff
@@ -108,6 +108,10 @@ commit_default = False
 
 By default, a script can be scheduled for execution at a later time. Setting `scheduling_enabled` to False disables this ability: Only immediate execution will be possible. (This also disables the ability to set a recurring execution interval.)
 
+### `rq_queue_name`
+
+This will override the standard `QUEUE_MAPPINGS` setting for the script to be process by the worker.  If the queue is not found then the default logic found in the settings file will be used.
```

Comment:
Maybe mention here that you will need to add any queues to your rq service start

Reply:
Done: https://github.com/netbox-community/netbox/pull/16552/commits/d34a87f5912c2073c5bc41db216f9435ab97b1dd